src/views/activity/activity_index_view.js

```diff
@@ -14,11 +14,15 @@ export default BaseView.extend({
   },
 
   afterRender () {
+    const startIndex = 1;
+
     this.removeSubviews();
-    let items = this.sliceCollection(this.adapter.results, ActivityResultsCollection, 1, 3);
+    let items = this.sliceCollection(this.adapter.results, ActivityResultsCollection, startIndex, 3);
     this.renderCollection(items, ActivityItemView, '.activity-results');
 
     // if there are results then show otherwise hide
-    this.adapter.results.length ? this.show() : this.hide(); // eslint-disable-line no-unused-expressions
+    this.adapter.results.length > startIndex ? this.show() : this.hide(); // eslint-disable-line no-unused-expressions
```

Comment:
We show the top activity item as a separate section, so we should only show this if there are 2 or more results, otherwise it's empty. I'll update the comment to explain that, it's confusion.